wiso.js

```diff
@@ -36,19 +36,21 @@
 
 
 function detectWeb(doc, url) {
-	if (url.search(/START=A[246]0/) !== -1) {
+	if (url.search(/document/) !== -1) {
 		if( ZU.xpath(doc, '//a[contains(@class, "boxExport")]').length>0 ) {
 			//single item --> generic fallback = journalArticle
 			return "journalArticle";
 		}
-	} else if ( ZU.xpath(doc, '//a/span[contains(@class, "boxHeader")]').length>0 ) {
+	} else if (getSearchResults(doc, true)) {
 		return "multiple";
 	}
 }
 
 
 function scrape(doc, url) {
-	var risUrl = url+"&OHNE_SAVE=1&T_TEMPLATE=ris";
+	var permaUrl = ZU.xpathText(doc, '//pre[strong]/text()');
```

Comment:
Ok, but, FYI, there are two text nodes there (one before the strong), so you may get a semicolon separating them (I don't think we filter out empty nodes). Doesn't matter for this use case though.

Regarding regexp, I don't think it's very specific, so if you can make it better, I would encourage it.